travis/travis_run_tests

```diff
@@ -56,12 +56,15 @@ if __name__ == '__main__':
     tests_unspecified = os.environ.get('TESTS') is None
     transifex_enabled = os.environ.get('TRANSIFEX') == '1'
     weblate_enabled = os.environ.get('WEBLATE') == '1'
-    is_oca_project = os.environ.get('TRAVIS_REPO_SLUG', '').startswith('OCA/')
+    is_oca_project = (os.environ.get('TRAVIS_REPO_SLUG', '') or
+                      os.environ.get('CI_COMMIT_REF_SLUG', ''))\
```

Comment:
You should use a nested `get`

`os.environ.get('TRAVIS_REPO_SLUG', os.environ.get('CI_COMMIT_REF_SLUG', ''))`